drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -102,4 +102,8 @@ config RX_MAX_DATA_SIZE
 	int "Maximum size of RX data"
 	default 1600
 
+config NRF700X_SCAN_LIMIT
+	int "Maximum number of scan results returned to application. Use negative values for unlimited scan results."
+	default -1
```

Comment:
What is the relation to configured heap size? E.g. what would be the max NRF700X_SCAN_LIMI with 25k heap size?

Reply:
We are still trying to profile the exact relationship between the NRF700X_SCAN_LIMIT and heap size. We do see that if the heap size is reduced below 20K the number of scan results displayed by the application start getting reduced. However with a 25K heap size we haven't seen a degradation w.r.t to the previous heap size of 100K.

Reply:
So how many scan results you've experimented with 25k heap size? And what happens if we run out of heap? I'll try to get this verified tomorrow with our application.

Reply:
We have seen up to approximately 110 APs. With lower heap sizes we have seen that in the same environment lesser number of results start getting displayed.

Reply:
Finding the correlation between max number of supported scans and heap size would be really cool. So that we can spare some memory if the scan limit is well known (i.e. it comes from the application use cases).

Reply:
I am also wondering much much limiting the number of scans impacts the overall scanning time. Do we have any estimation here?

Reply:
I guess it shouldn't affect on the scanning time because you need to find everything in order to be able to sort the  APs with RSSI.

Reply:
Fair point, so it's not like we first define the the limit and then do the sorting on the subset of randomly propagated results, which totally makes sense. @sachinthegreen is this assumption correct?

Reply:
Yes. The scan times remain the same.